book/reference/glossary.md

```diff
@@ -68,19 +68,14 @@
   meant for technical documentation and publishing. It is used by Jupyter Book.
 
 [Slack](https://slack.com)
-  A communication platform that we use to share information. We use separate channels
-  for each project and also rely on the video features. If possible we recommend
-  [downloading the Slack app](https://slack.com/downloads). If your agency does not allow
-  you to use the app, you can still interface with Slack in a web browser.
+  A communication platform that we use to share information. We use separate channels for each project and also rely on the video features. If possible we recommend [downloading the Slack app](https://slack.com/downloads). If your agency does not allow you to use the app, you can still interface with Slack in a web browser.
 
 ```
 
 ## Topical (event specific)
+
 ```{glossary}
 [ICESat-2](https://icesat-2.gsfc.nasa.gov)
-  Ice, Cloud, and land Elevation Satellite (2).
-  Launched in September 2018, the ICESat-2 satellite continues the ICESat and IceBridge missions to
-  collect multi-year elevation data.
-  Mission science goals focus on assessing changing land and sea ice and forest heights.
+  Ice, Cloud, and land Elevation Satellite (2). Launched in September 2018, the ICESat-2 satellite continues the ICESat and IceBridge missions to collect multi-year elevation data. Mission science goals focus on assessing changing land and sea ice and forest heights.
 
   ```
```

Comment:
This should at least fix the `/home/runner/work/CryoCloudWebsite/CryoCloudWebsite/book/reference/glossary.md:38: WARNING: glossary term must be preceded by empty line` warning.

```suggestion
```
```

Reply:
I thought I'd done that glossary change and it hadn't improve the warning last night. I think either all of the entries need adjustment, or there is something wrong with the entry I added to the glossary for 2i2c which is at line 38. 

> Let me see if there's a workaround.

Ok I looked into this last night, but wasn't sure where to apply it:
https://myst-parser.readthedocs.io/en/latest/syntax/optional.html?highlight=anchor#auto-generated-header-anchors

It says to add <myst_heading_anchors = 3> to conf.py, and it is related to sphinx. So I was thinking it goes in our _config.yml file maybe under parse or sphinx?

Reply:
And here: https://myst-parser.readthedocs.io/en/latest/syntax/syntax.html#markdown-links-and-referencing

Reply:
This glossary change will only fix 1/8 warnings. After some reading, the problem seems to be on the `jupyter-book` not using a newer version of `myst-parser`, this should be fixed in https://github.com/executablebooks/jupyter-book/pull/1842.

In the meantime, we could temporarily remove that `--warningsiserror` flag in [CryoCloudWebsite/scripts/build_resources.sh](https://github.com/CryoInTheCloud/CryoCloudWebsite/blob/5b50d8c6ddf821f68a6ca02fc90c7292d05e92d0/scripts/build_resources.sh#L20).

Reply:
Ah I see. Great. I will work on that.